Hush/Assets/Scenes/Menus/MainMenu.unity

```diff
@@ -0,0 +1,4942 @@
+%YAML 1.1
```

Comment:
For consistency, keep the buttons font for buttons. (you can use the buttons font for text here if you want, but not the other way around)


### minor
1. How many save files per "install" will we support? Just one? Mutliple? The current interface for it feels a bit off. Either we make save files A, B C or we just have a single file? eiother way, here's a mockup
![image](https://user-images.githubusercontent.com/35614138/143965011-cdd6bd27-ff23-4da1-88d3-95358e6ad503.png)
 
2. add a black panel w/ opacity (like alpha = 75% ish ) dim down the background and make this menu stand out more. 
![image](https://user-images.githubusercontent.com/35614138/143964089-513be39e-49d3-46da-b846-3fdeb770c9a9.png)

Reply:
i'm not super clear on that either. from what i understood it was just a single one (wherein a `Start` vs `Continue` might make more sense, if that's what you mean) but I don't know if we're actually considering multiple save files. if we want multiple save files we can make task for it and implement it in another PR